policy-supplemental/platforms.md

```diff
@@ -20,6 +20,7 @@
 | vms-ia64-p32 |        | OpenVMS 8.4 |        | ia64         |        | VSI C 7.4 [^1]  |        | \@levitte              |
 | vms-ia64-p64 |        | OpenVMS 8.4 |        | ia64         |        | VSI C 7.4 [^2]  |        | \@levitte              |
 | vms-x86\_64  |        | OpenVMS 8.4 |        | x86\_64      |        | VSI C X7.4 [^3] |        | \@levitte              |
+| linux-aarch64 |       | Linux       |        | AArch64      |        | gcc             |        | \@tom-cosgrove-arm     |
```

Comment:
I wonder if it is worth adding an additional column of "other maintainers". We are losing some useful information here.

Reply:
(and remind me: additional commit, or force push with such an update?)

Reply:
> Would a column "Nominated Community Maintainers" make sense?

That works for me

> (and remind me: additional commit, or force push with such an update?)

I'd do it as a fixup commit. But either way.